docs/aerodromes/classc/Edinburgh.md

```diff
@@ -4,7 +4,74 @@
 
 --8<-- "includes/abbreviations.md"
 
-Reserved.
+## Positions
 
-!!! note
-    Additional charts to the AIP may be found in the RAAF TERMA document, available towards the bottom of [RAAF AIP page](https://ais-af.airforce.gov.au/australian-aip){target=new}
\ No newline at end of file
+| Name            | Callsign           | Frequency  | Login Identifier |
+| --------------- | ------------------ | ---------- | ---------------- |
+| Edinburgh ADC   | Edinburgh Tower    | 118.300    | ED_TWR           |
+| Edinburgh SMC   | Edinburgh Ground   | 134.100    | ED_GND           |
+
+## Airspace
+<figure markdown>
+![EDN ADC Airspace](img/edncircuits.png){ width="700" }
+</figure>
+
+- **EDN ADC Airspace**: `SFC` to `A015` within the Edinburgh Control Zone (ED CTR).
```

Comment:
Seems to be the case that they control to A015, but not 100% sure

Reply:
Confirmed by DAH